tests/queries/0_stateless/02353_isnullable.sql

```diff
@@ -3,3 +3,9 @@ SELECT isNullable(toNullable(3));
 
 SELECT isNullable(NULL);
 SELECT isNullable(materialize(NULL));
+
```

Comment:
I found that it's very helpful to use `-- { echoOn }`, - it's much easier to relate output in reference file and queries

Reply:
Yeah, I know about it, but don't think it's reasonable to use in such small tests